scraper.go

```diff
@@ -143,19 +145,17 @@ type stateJson struct {
 type Scraper struct {
 	Sources       []*LogSource
 	Hostname      string
+	PCname        string
```

Comment:
Same name everywhere. Not PCname. Stick to ownhostname. In this case, to be specific, it should be OwnHostname.